sharding-jdbc/sharding-jdbc-orchestration/src/test/resources/META-INF/services/org.apache.shardingsphere.orchestration.center.api.ConfigCenter

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 
-org.apache.shardingsphere.orchestration.temp.internal.registry.fixture.FifthTestRegistryCenter
-org.apache.shardingsphere.orchestration.temp.internal.registry.fixture.ThirdTestRegistryCenter
-org.apache.shardingsphere.orchestration.temp.internal.registry.fixture.ForthTestRegistryCenter
+org.apache.shardingsphere.shardingjdbc.orchestration.util.TestRegistryCenter
```

Comment:
As for ConfigCenter, shall these four TestRegistryCenters be renamed as TestConfigCenter?